testdata/google/events/cloud/audit/v1/LogEntryData-bigqueryjobcompleted.json

```diff
@@ -0,0 +1,92 @@
+{
+  "data": {
```

Comment:
As Jon mentioned, put just the _contents_ of the `"data"` property as the JSON object for this file.